nautobot/core/templatetags/helpers.py

```diff
@@ -376,10 +376,9 @@ def format_time_zone(value):
     """
     timezone = value
     offset = tzoffset(value)
-    # TODO: This should be but it is not working for some reason
-    # {% timezone "{}" %}{% now "SHORT_DATETIME_FORMAT" %}{% endtimezone %}
+    current_time = datetime.datetime.now(timezone).strftime("%Y-%m-%d %I:%M %p")
```

Comment:
Downside of this approach is that it doesn't respect the configured `setttings.SHORT_DATETIME_FORMAT`, right?

Reply:
Could revert this for now, given it's specific to Location?